src/components/forms/create-question-dialog.tsx

```diff
@@ -12,10 +14,38 @@ import { Input } from "@/components/ui/input";
 import { Button } from "../ui/button";
 import { Label } from "../ui/label";
 import { PlusIcon } from "lucide-react";
-
 import { Textarea } from "../ui/textarea";
+import createQuestionData from "@/utils/questionApiClient";
 
 export function CreateQuestionDialog() {
+  const [name, setName] = useState<string>(""); // Zustand für den Namen
+  const [description, setDescription] = useState<string>(""); // Zustand für den Benutzer-ID
+  const [min, setMinimum] = useState<string>("");
+  const [steps, setStep] = useState<number | undefined>(undefined);
+  const [max, setMaximum] = useState<string>("");
+  const [surveyId, setSurveyId] = useState<string>("");
+
+  const handleSaveClick = async () => {
+    // Standardwert für steps, wenn es undefined ist
+    const stepsValue = steps !== undefined ? steps : 0;
+    try {
+      // Rufe die createQuestionData-Funktion auf, um die Frage zu erstellen
+      const questionData = await createQuestionData(
+        name,
+        description,
+        min,
+        stepsValue,
+        max,
+        surveyId
```

Comment:
Where does the surveyId come from? You should read it from the page and pass it to this component as a property